### ISSUE TITLE
Please provide debug symbols packages for all ROS packages with binary content

### ISSUE THREAD

**Issue by nayomal** [ros-infrastructure/bloom#373]
Most Debian packages with binary content provide a separate debug symbols package. The lack of debug symbols for ROS packages makes it very painful to debug issues when failures/crashes occur within ROS. We often have to manually build the shared object/ executable and replace the installed version with those or download the source and debian build files, patch it to generate a debug package, build and then install the new package and debug symbols package. This takes even more time and effort if a particular issue spans across multiple ROS libraries where we'd have to rebuild several packages before we can start debugging effectively. If debug packages for ROS were available it would just be a matter of running "sudo apt-get install <package for which debug symbols are needed>-dbg" and then running the debugger.

**Comment by wjwwood** [comment]
Duplicate of https://github.com/ros-infrastructure/bloom/pull/32

**Comment by wjwwood** [comment]
Sorry, but we haven't had time to address this. Even if we fixed it right now, we'd have to wait for new releases (or re-releases) of all the ROS packages. We could manually modify each control file before building if the dbg packages had not been generated, but I don't think we have time to implement this right now. You can push on #32 if you'd like to champion the issue.

**Comment by nayomal** [comment]
Thanks William! Will follow up on the other ticket. Sorry for opening a duplicate. I missed #32 as it was a merge request and GitHub had restricted my search to issues.
